src/main/resources/migrations.xml

```diff
@@ -1189,4 +1189,48 @@
 		<addUniqueConstraint tableName="savedSearches" columnNames="ownerId, name" constraintName="uniqueOwnerIdName" />
 	</changeSet>
 
+	<changeSet id="rename-parent-task-in-tasks" author="maradragan" dbms="!sqlite">
+		<renameColumn
+			columnDataType="int"
+			oldColumnName="parentTaskId"
+			newColumnName="customFieldRef1Id"
+			tableName="tasks" />
+		<dropForeignKeyConstraint baseTableName="tasks" constraintName="fk_tasks_parent"/>
```

Comment:
Strictly speaking, the `dropForeignKeyConstraint` should come _before_ the `renameColumn`.